<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseLineWidget</class>
 <widget class="QWidget" name="BaseLineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxLineNum">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>500</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>基准线数量</string>
   </property>
   <widget class="QPushButton" name="btnLineSub">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLineAdd">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNextLine">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPreLine">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtCurrentLine">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>51</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="txtLineNum">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>51</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>51</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxCoordinate">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>500</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>坐标微调</string>
   </property>
   <widget class="QPushButton" name="btnMoveRight">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveDown">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveLeft">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveUp">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Point1">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>48</y>
      <width>52</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>点a:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtPoint1Coor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>48</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(320,0)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Point2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>111</y>
      <width>52</width>
      <height>25</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>点b:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtPoint2Coor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>111</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(320,480)</string>
    </property>
   </widget>
   <zorder>btnMoveUp</zorder>
   <zorder>btnMoveRight</zorder>
   <zorder>btnMoveDown</zorder>
   <zorder>btnMoveLeft</zorder>
   <zorder>radioButton_Point1</zorder>
   <zorder>txtPoint1Coor</zorder>
   <zorder>radioButton_Point2</zorder>
   <zorder>txtPoint2Coor</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
